sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala

```diff
@@ -64,7 +64,7 @@ import org.apache.spark.util.CircularBuffer
  * This does not include exceptions thrown during the eager execution of commands, which are
  * grouped into [[QueryCompilationErrors]].
  */
-object QueryExecutionErrors {
+private[spark] object QueryExecutionErrors {
```

Comment:
cc @karenfeng

Reply:
shall we mark the entire `org.apache.spark.sql.errors` as private?

See https://github.com/apache/spark/blob/master/project/SparkBuild.scala#L944-L973

Reply:
SGTM. @karenfeng will there be any new public API under `org.apache.spark.sql.errors`?

Reply:
cc @allisonwang-db

I think that the only useful API to expose related to this may be `SparkThrowable`, which contains `errorClass` and `sqlState`. But that is under `org.apache.spark`.

Reply:
LGTM

Reply:
ok, let me mark org.apache.spark.sql.errors as private